db/seeds.rb

```diff
@@ -156,17 +104,84 @@
 
 puts "They decided"
 
-puts "Create the Rulebooks..."
+puts "Creating the Rulebooks..."
 
 BOOK_TITLES.each_with_index do |book, index|
   genre = Genre.find_by(name: book[:genre])
   rulebook = Rulebook.create!(
     name: book[:title],
     description: book[:description],
-    genre: genre,
     img_url: book[:img]
     )
+  Pairing.create!(
+    genre: genre,
+    rulebook: rulebook
+    )
   puts "#{index + 1}. #{rulebook.name} published"
 end
 
-puts "All Rulesbooks created"
+puts "All Rulebooks created"
+
+puts "Generating Groups..."
+group_num = 0
+
+2.times do
+  preference = gandalf.preferences.sample.genre
```

Comment:
where do you use this variable?

Reply:
This is planned to be used to grab a rulebook that matches the user's preference